pages/login.page.js

```diff
@@ -0,0 +1,38 @@
+import BasePage from "./base.page";
+const PAGE_TIMEOUT = 10000;
+const BASE_PAGE = BasePage;
+
+class LoginPage {
+    
+    get usernameInput () { return $('#username'); }
+    get passwordInput () { return $('#password'); }
+    get rememberMe () { return $('#remember-me'); }
+    get logInButton () { return $('#submit'); }
+    get failedLoginMessage () {return $('[class= "badge badge-pill badge-danger"]'); }
+
+    async navigateToPage(){
+        await BASE_PAGE.open('login');
+    }
+    
+    async completeCredentials(username, password) {
+        await BASE_PAGE.completeFiels(await this.usernameInput, username);
+        await BASE_PAGE.completeFields(await this.passwordInput, password);
+      }
+
+    async logInWithEnter(username, password){
+        this.completeCredentials(username, password);
+        await this.logInButton.keys('Enter');
```

Comment:
Prefer implementing a method in BasePage that presses enter on some element, with similar interface than your current `clickElement(element)`, and using it here instead of raw key sending, as a means to remain at the same level of abstraction throughout the whole method. Currently you are mixing 2 levels: credentials handling and web element's direct method calling.